### DISCUSSION TITLE
Where's config file on Miyoo Mini

### DISCUSSION THREAD

**Discussion by okarowarrior** [diasurgical/DevilutionX#5644] [Q&A] [answered]
I'm playing the port of Diablo Hellfire on my Miyoo mini (linux based hanheld). It runs flawlessly, but the settings dont save each time I close the game, so I have to manually set everything each time I open it and It's a hassle.
I see that on the wiki there are listed some paths for the main OS's, but I fail to find the config file the Miyoo uses so I manually set what I want. 
Thanks in advance

**Comment by glebm** [comment, marked as answer]
Looks like the launcher script needs to `mkdir` the config directory.

**Reply by glebm** [reply]
As a workaround, you can create the directory yourself and the settings will save correctly

For OnionOS:
https://github.com/diasurgical/devilutionX/blob/7a1f962e33b737bfb60d91ecec8da4ee93d3a52a/Packaging/miyoo_mini/skeleton_OnionOS/Roms/PORTS/Binaries/Diablo.port/launch.sh#L5

**Reply by okarowarrior** [reply]
Thanks for taking your precious time to reply to me, glebm. I will try it out asap!

**Reply by glebm** [reply]
The latest build creates the directories automatically: https://github.com/diasurgical/devilutionX/actions/runs/3900547553

(https://github.com/diasurgical/devilutionX/pull/5650)
